src/components/Panel/Actions/ActionDetails.tsx

```diff
@@ -0,0 +1,126 @@
+import { Currency, InformationIcon } from "@/components";
+import type { OracleQueryUI } from "@/types";
+import styled from "styled-components";
+import { Link, Text } from "../style";
+
+export function ActionDetails({
+  bond,
```

Comment:
Very confused by putting "action" in all these components. Action should be reserved for things user can do to mutate something, but many of these seem just for display

Reply:
This component is called action details because its part of the actions component (it shows the text about the actions). But since this component is already in a directory called Actions, I renamed it to be just `Details` instead to avoid confusion. Are there other components that also have confusing names? The only other one with action in the name is `Actions`, which is the component that holds all the actions the user can take with the contract. I think that name makes sense.